### PR TITLE
FAQ - update 'access_list' entry

### PR DESCRIPTION
**Pull request by heinezen**
Changelog:
- "Some firewalls" -> "some firewall settings" (english/german)
- "contacting the backend" -> "contacting its test result/verification backend" (english/german)
- format access list as HTML list

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@heinezen 
If you are updating the https://www.coronawarn.app/en/faq/#access-list entry, please consider also the open issue https://github.com/corona-warn-app/cwa-website/issues/952 regarding 
https://github.com/corona-warn-app/cwa-website/issues/952#issuecomment-791924528 and

- *.rki.de
- *.howsmyssl.com

linked to Internal Tracking ID: [EXPOSUREAPP-1906](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-1906).

File: src/data/faq_de.json [hunk 1]
```diff
@@ -102,7 +102,8 @@
                         "anchor": "access-list",
                         "active": true,
                         "textblock": [
-                            "Einige Firewalls verhindern möglicherweise den Zugriff der CWA auf das Backend. Das kann in einigen Fällen zu einer Fehlermeldung führen, unter anderem 'URSACHE: 4000, error during web request, HTTP status 901'. Sollten sie eine Firewall benutzen (Router oder App), vergewissern sie sich, dass die folgenden Domains in der Access-Liste für Web- und den DNS-Requests berücksichtigt sind:<br/>*.coronawarn.app<br/>*.t-online.de"
+                            "Einige Firewalleinstellungen verhindern möglicherweise den Zugriff der CWA auf das Corona-Warn-App-Backend. Das kann in einigen Fällen zu einer Fehlermeldung führen, unter anderem 'URSACHE: 4000, error during web request, HTTP status 901'. Sollten sie eine Firewall benutzen (Router oder App), vergewissern sie sich, dass die folgenden Domains in der Access-Liste für Web- und den DNS-Requests berücksichtigt sind:",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:105]: In "Sollten **sie** eine Firewall benutzen (Router oder App), vergewissern **sie** sich, dass die folgenden Domains in der Access-Liste für Web- und den DNS-Requests berücksichtigt sind:"

change both occurrences of "sie" to "Sie".

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Two typos to correct which were there before (sie => Sie) and a request to incorporate the additional domains noted in #952.

File: src/data/faq_de.json (hunk 1 above)
Comment (heinezen) [reply src/data/faq_de.json]: fixed

**Comment by heinezen** [comment]
> 
> 
> @heinezen
> If you are updating the https://www.coronawarn.app/en/faq/#access-list entry, please consider also the open issue #952 regarding
> [#952 (comment)](https://github.com/corona-warn-app/cwa-website/issues/952#issuecomment-791924528) and
> 
>     * *.rki.de
> 
>     * *.howsmyssl.com
> 
> 
> linked to Internal Tracking ID: [EXPOSUREAPP-1906](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-1906).

I included the subdomains for URLs, so that the firewall settings do not have to be as lax. Did you close #952 by accident?

**Comment by MikeMcC399** [comment]
@heinezen 
> Did you close #952 by accident?

It seems that I did! Thank you for re-opening.

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
image

something broken... will check

**Comment by dsarkar** [comment]
image

image


LGTM
Thanks to everybody involved!
@heinezen @Ein-Tim @MikeMcC399 .... and others

File: src/data/faq.json [hunk 2]
```diff
@@ -103,7 +103,10 @@
                         "anchor": "access-list",
                         "active": true,
                         "textblock": [
-                            "Some firewalls might prevent the CWA from contacting the backend. This might lead to error notifications, including 'CAUSE: 4000, error during web request, HTTP status 901'. In case you make use of a firewall (router or app), make sure to include the following domains in the access-list for web requests as well as for DNS requests:<br/>*.coronawarn.app<br/>*.t-online.de"
+                            "Some firewall settings might prevent the CWA from contacting the Corona-Warn-App backend. This might lead to error notifications, including 'CAUSE: 4000, error during web request, HTTP status 901'. In case you make use of a firewall (router or app), make sure to include the following domains in the access-list for web requests as well as for DNS requests:",
+                            "<ul><li>*.coronawarn.app</li><li>*.t-online.de</li><ul>",
```
Comment (MikeMcC399) [inline src/data/faq.json]: </li><ul> => </li></ul>

File: src/data/faq.json [hunk 3]
```diff
@@ -103,7 +103,10 @@
...
                         "textblock": [
-                            "Some firewalls might prevent the CWA from contacting the backend. This might lead to error notifications, including 'CAUSE: 4000, error during web request, HTTP status 901'. In case you make use of a firewall (router or app), make sure to include the following domains in the access-list for web requests as well as for DNS requests:<br/>*.coronawarn.app<br/>*.t-online.de"
+                            "Some firewall settings might prevent the CWA from contacting the Corona-Warn-App backend. This might lead to error notifications, including 'CAUSE: 4000, error during web request, HTTP status 901'. In case you make use of a firewall (router or app), make sure to include the following domains in the access-list for web requests as well as for DNS requests:",
+                            "<ul><li>*.coronawarn.app</li><li>*.t-online.de</li><ul>",
+                            "If you want to allow the usage of the (optional) survey system, you should additionally add these domains to your access-list:",
+                            "<ul><li>befragungen.rki.de</li><li>www.howsmyssl.com</li><ul>"
```
Comment (MikeMcC399) [inline src/data/faq.json:109]: </li><ul> => </li></ul>

File: src/data/faq_de.json [hunk 4]
```diff
@@ -102,7 +102,10 @@
                         "anchor": "access-list",
                         "active": true,
                         "textblock": [
-                            "Einige Firewalls verhindern möglicherweise den Zugriff der CWA auf das Backend. Das kann in einigen Fällen zu einer Fehlermeldung führen, unter anderem 'URSACHE: 4000, error during web request, HTTP status 901'. Sollten sie eine Firewall benutzen (Router oder App), vergewissern sie sich, dass die folgenden Domains in der Access-Liste für Web- und den DNS-Requests berücksichtigt sind:<br/>*.coronawarn.app<br/>*.t-online.de"
+                            "Einige Firewalleinstellungen verhindern möglicherweise den Zugriff der CWA auf das Corona-Warn-App-Backend. Das kann in einigen Fällen zu einer Fehlermeldung führen, unter anderem 'URSACHE: 4000, error during web request, HTTP status 901'. Sollten Sie eine Firewall benutzen (Router oder App), vergewissern Sie sich, dass die folgenden Domains in der Access-Liste für Web- und den DNS-Requests berücksichtigt sind:",
+                            "<ul><li>*.coronawarn.app</li><li>*.t-online.de</li><ul>",
```
Comment (MikeMcC399) [inline src/data/faq_de.json]: `</li><ul>` => `</li></ul>`

File: src/data/faq_de.json [hunk 5]
```diff
@@ -102,7 +102,10 @@
...
+                            "Einige Firewalleinstellungen verhindern möglicherweise den Zugriff der CWA auf das Corona-Warn-App-Backend. Das kann in einigen Fällen zu einer Fehlermeldung führen, unter anderem 'URSACHE: 4000, error during web request, HTTP status 901'. Sollten Sie eine Firewall benutzen (Router oder App), vergewissern Sie sich, dass die folgenden Domains in der Access-Liste für Web- und den DNS-Requests berücksichtigt sind:",
+                            "<ul><li>*.coronawarn.app</li><li>*.t-online.de</li><ul>",
+                            "Falls Sie zusätzlich die Nutzung der (optionalen) Datenspende-Funktion erlauben wollen, sollten Sie zusätzlich diese Domains zu Ihrer Access-Liste hinzufügen:",
+                            "<ul><li>befragungen.rki.de</li><li>www.howsmyssl.com</li><ul>"
```
Comment (MikeMcC399) [inline src/data/faq_de.json:108]: `</li><ul>` => `</li></ul>`

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Lists are improperly terminated in HTML.

**Review by MikeMcC399 — COMMENTED** [review]
Comments withdrawn. You already fixed it!

**Pull request merged**